wagtail/wagtailcore/urls.py

```diff
@@ -18,9 +20,16 @@
     serve_pattern = r'^([\w\-/]*)$'
 
 
+WAGTAIL_FRONTEND_LOGIN_TEMPLATE = getattr(
+    settings, 'WAGTAIL_FRONTEND_LOGIN_TEMPLATE', 'wagtailcore/login.html'
+)
+
+
 urlpatterns = [
     url(r'^_util/authenticate_with_password/(\d+)/(\d+)/$', views.authenticate_with_password,
         name='wagtailcore_authenticate_with_password'),
+    url(r'^_util/login/$', auth_views.login, {'template_name': WAGTAIL_FRONTEND_LOGIN_TEMPLATE},
```

Comment:
I think that's right, yep. (I didn't cover this in the docs, as it's the standard django.contrib.auth functionality and is covered in Django's own docs.)